pkgs/top-level/python-packages.nix

```diff
@@ -9428,6 +9428,13 @@ self: super: with self; {
 
   pyvisa-py = callPackage ../development/python-modules/pyvisa-py { };
 
+  pyvista = callPackage ../development/python-modules/pyvista {
+    vtk = toPythonModule (pkgs.vtk_9.override {
```

Comment:
We cannot do this because it will create issues when using together with another vtk package in a python env.

Reply:
Opened #208223 to bump python3Packages.vtk.